src/components/Cart/Cart.jsx

```diff
@@ -70,6 +70,10 @@ const Cart = ({
                       onSelect={() => {}}
                       disabled
                     />
+                    <p>
+                      Locker #{rest.lockerId?.lockerNumber} –{' '}
+                      {rest.lockerId?.location}
+                    </p>
```

Comment:
You can extract this into a constant or function, so we will not see logic inside the jsx